docs/api/carts.rst

```diff
@@ -0,0 +1,810 @@
+
+        ],
+        "adjustments_total":0,
+        "total":0,
+        "customer":{
+            "id":1,
+            "email":"shop@example.com",
+            "first_name":"John",
+            "last_name":"Doe",
+            "user":{
+                "id":1,
+                "username":"shop@example.com",
+                "username_canonical":"shop@example.com"
+            }
+        },
+        "channel":{
+            "code":"US_WEB",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/channels\/1"
+                }
+            }
+        },
+        "currency_code":"USD",
+        "checkout_state":"cart"
+    }
+
+.. note::
+    A currency code will be added automatically based on a channel settings. `Read more about channels`__
+
+__ http://docs.sylius.org/en/latest/book/configuration/channels.html
+
+If you try to create a resource without name or code, you will receive a 400 error.
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/carts/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X POST
+
+Example Response
+^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 400 Bad Request
+
+.. code-block:: json
+
+    {
+        "code":400,
+        "message":"Validation Failed",
+        "errors":{
+            "children":{
+                "customer":{
+                    "errors":[
+                        "This value should not be blank."
+                    ]
+                },
+                "localeCode":{
+                    "errors":[
+                        "This value should not be blank."
+                    ]
+                },
+                "channel":{
+                    "errors":[
+                        "This value should not be blank."
+                    ]
+                }
+            }
+        }
+    }
+
+Collection of Carts
+-------------------
+
+To retrieve the paginated cart list you will need to call the ``/api/v1/carts/`` endpoint with ``GET`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/carts/
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| limit         | query          | *(optional)* Number of items to display per page, by default = 10 |
```

Comment:
Since this is for `cart` specifically, wdyt about changing `Number of items to display per page` to `Number of carts displayed per page`?